plugin/api.thrift

```diff
@@ -130,6 +130,35 @@ struct Argument {
      * Argument type.
      */
     2: required Type type
+    /**
+     * Annotations defined on this argument.
+     *
+     * Note that these are the Thrift annotations listed after the 
+     * parameter or exception name.
+     *
+     * Given,
+     *
+     *   void foo(
+     *     1: string bar (foo = "bar", validate)
+     *   ) throws (
+     *     1: BazError fail (bar = "foo")
+     *   )
+     *
+     * The annotations for the first argument will be,
+     *
+     *   {
+     *     "foo": "bar",
+     *     "validate": "",
+     *   }
+     *
+     * and the annotations for BazError will be,
```

Comment:
```suggestion
     * Similarly, the annotations for BazError will be,
```

(It's preferable to start a new sentence here rather than try to continue the previous after the code block.)